docker-compose.yaml

```diff
@@ -0,0 +1,39 @@
+services:
+  kt-app:
+     image: jakefrosty/korotagger:1.3
+     cpus: 0.5
+     mem_limit: 100m
+     container_name: korotagger-app
+     environment:
+       yt_api_key: your-yt-api-key
+       holodex_api_key: your-holodex-api-key
+       POSTGRES_HOST: kt-db
```

Comment:
is there a reason to split this out into multiple environment variables over providing an example for how to format a postgres url?

there doesn't seem to be any handling for special characters or such so this would presumably have the same pitfalls, just less transparently

Reply:
The community docker-compose files I'm used to have always used split database credentials and I didn't wanna make it different